### ISSUE TITLE
Add a regex validator (AllowedPattern) to all of the *SSLCertificateARN fields

### ISSUE THREAD

**Issue by irvingpop** [chef-customers/aws_native_chef_server#43]
As a user, I would like immediate feedback if I've put an obvious typo in my SSLCertificateARN field.

**Comment by irvingpop** [comment]
one of these maybe?

```yaml
AllowedPattern: ^arn:aws:acm:([a-zA-Z0-9_\-\:\/]+)$
AllowedPattern: ^arn:aws:acm:[a-zA-Z0-9_\-\:\/]*$
```
maybe?

**Comment by irvingpop** [comment]
this is fixed by PR #50
